src/test/java/uk/nhs/prm/deduction/e2e/models/ContinueRequestMessage.java

```diff
@@ -0,0 +1,49 @@
+package uk.nhs.prm.deduction.e2e.models;
+
+import uk.nhs.prm.deduction.e2e.utility.Resources;
+
+import java.util.UUID;
+
+public final class ContinueRequestMessage {
+    private UUID conversationId;
+    private UUID messageId;
+    private String sourceGpOds;
+    private String destinationGpOds;
+    private String sourceGpAsid;
+    private String destinationGpAsid;
+
+    public UUID getConversationId() {
+        return conversationId;
+    }
+
+    public UUID getMessageId() {
+        return messageId;
+    }
+
+    public ContinueRequestMessage(UUID conversationId, UUID messageId, String sourceGpOds, String destinationGpOds, String sourceGpAsid, String destinationGpAsid) {
+        this.conversationId = conversationId;
+        this.messageId = messageId;
+        this.sourceGpOds = sourceGpOds;
+        this.destinationGpOds = destinationGpOds;
+        this.sourceGpAsid = sourceGpAsid;
+        this.destinationGpAsid = destinationGpAsid;
+    }
+
+    public String toJsonString() {
+        return Resources.readTestResourceFile("COPC_IN000001UK01")
+                .replaceAll("DBC31D30-F984-11ED-A4C4-956AA80C6B4E", conversationId())
+                .replaceAll("DE304CA0-F984-11ED-808B-AC162D1F16F0", messageId())
+                .replaceAll("B85002", sourceGpOds)
+                .replaceAll("200000001613", sourceGpAsid)
+                .replaceAll("M85019", destinationGpOds)
+                .replaceAll("200000000149", destinationGpAsid);
+    }
+
+    public String conversationId() {
```

Comment:
Hi, I have added the static method getUuidAsUpperCaseString to TestUtils class and call it for conversation ids and message ids. 
Later could you have a look and see if I did it right? 
Because conversation id is a private field so I keep the getter methods, but now all those getter would call to our getUuidAsUpperCaseString :)

Reply:
Sorry I overlooked this comment...
For `conversationId()`, in the e2e test that is a part that we need the conversation id from the message (the test use the conversation id to find an expected message in outbound queue), so I have kept the getter method for this purpose.